### PR TITLE
Fix API production refuses to handle localhost

### PR DESCRIPTION
**Pull request by ketphan02**

### CODE REVIEW

File: api/api/settings/prod.py [hunk 1]
```diff
@@ -94,7 +94,7 @@
 ]
 
 # CORS
-CORS_ALLOWED_ORIGINS = ["*"]
+CORS_ALLOW_ALL_ORIGINS = True
```
Comment (Opeyem1a) [inline api/api/settings/prod.py:97]: On prod?

**Review by Opeyem1a — CHANGES_REQUESTED** [review]

File: requirements.txt [hunk 2]
```diff
@@ -10,6 +10,7 @@ async-lru==2.0.4
 attrs==23.1.0
 Babel==2.13.1
 backcall==0.2.0
+bcrypt==4.1.2
```
Comment (Opeyem1a) [inline requirements.txt:13]: also why?

**Review by Opeyem1a — CHANGES_REQUESTED** [review]

File: api/api/settings/prod.py (hunk 1 above)
Comment (ketphan02) [reply api/api/settings/prod.py:97]: Yes, since this is public API

File: requirements.txt (hunk 2 above)
Comment (ketphan02) [reply requirements.txt:13]: I save the encrypted key as a password, therefore, even when the key is leaked (not likely), it is less likely to exploit our service

**Review by Opeyem1a — APPROVED** [review]

**Pull request merged**